dynamic-simulation/dynamic-simulation-tool/src/main/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationTool.java

```diff
@@ -45,6 +48,7 @@
 public class DynamicSimulationTool implements Tool {
 
     private static final String CASE_FILE = "case-file";
+    private static final String MAPPING_FILE = "mapping-file";
```

Comment:
we should also change this option's name

Reply:
Done